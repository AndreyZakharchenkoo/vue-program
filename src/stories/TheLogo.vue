<template>
  <router-link
    to="/"
    class="logo"
    aria-label="Logo">
    <b>netflix</b>roulette
  </router-link>
</template>

<script>
export default {
  name: 'TheLogo',
};
</script>

<style scoped>
.logo {
  color: #F65261;
  font-size: 20px;
  font-weight: 500;
  text-decoration: none;
}
</style>
