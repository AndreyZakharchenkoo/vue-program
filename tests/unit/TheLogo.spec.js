import { shallowMount } from '@vue/test-utils';
import TheLogo from '@/stories/TheLogo.vue';

describe('TheLogo.vue', () => {
  it('Displays default logo text', () => {
    const wrapper = shallowMount(TheLogo, {
      stubs: ['router-link', 'router-view'],
    });
    expect(wrapper.text()).toContain('netflix');
  });
});
