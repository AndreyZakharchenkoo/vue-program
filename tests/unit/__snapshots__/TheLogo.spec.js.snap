// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheLogo.vue Renders correctly 1`] = `
<router-link-stub
  aria-label="Logo"
  class="logo"
  to="/"
>
  <b>
    netflix
  </b>
  roulette

</router-link-stub>
`;
